apps/web-client/src/lib/db/cr-sqlite/__tests__/supplier-order.test.ts

```diff
@@ -108,8 +109,73 @@ describe("Supplier order handlers should", () => {
 		const newPossibleOrderLines = await getPossibleSupplierOrderLines(db, 1);
 		expect(newPossibleOrderLines.length).toBe(0);
 	});
+	it("Aggregates supplier orders lines quantity when placing an order", async () => {
```

Comment:
This description should align with the behaviour: "... when getting possible order lines"